logo.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="-149 451 100 100" style="enable-background:new -149 451 100 100;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#DBC01D;}
+	.st1{fill:#383307;}
+</style>
+<g id="hexagon_2_">
+	<g id="hexagon_1_">
+		<path class="st0" d="M-63.7,527l-30.5,17.6c-3,1.8-6.6,1.8-9.7,0l-30.5-17.6c-3-1.8-4.8-5-4.8-8.4v-35.2c0-3.5,1.8-6.6,4.8-8.4
+			l30.5-17.6c3-1.8,6.6-1.8,9.7,0l30.5,17.6c3,1.8,4.8,5,4.8,8.4v35.2C-58.8,522.1-60.6,525.2-63.7,527z"/>
+	</g>
+</g>
+<g id="dom">
+	<path class="st1" d="M-105.6,473.2c0,0,0.1,5.7-2.2,12.3c-1.6,4.7-3.8,7.1-9,12.5c-2.5,2.5-9.5,8.7-14.8,15.4
+		c-4.1,5.2-6.5,9.7-6.4,9.7c0,0,0.9,2.3,3.7,4c2.2,1.3,7.4,4.3,7.4,4.3s2.1-3.6,3.4-5.6c1.5-2.3,4-5.4,4-5.4l8,12.2l11.5-7.7
+		l-9.7-14.5c0,0,4.6-4.4,5.8-5.8c1.2-1.3,3.2-3.7,3.2-3.7s4.9,2.9,6.8,4.2c2.8,1.8,6.4,4.7,6.4,4.7l-10.1,15.1l11.5,7.7l8.4-12.5
+		c0,0,3.6,5.1,4.7,6.8s2.3,4.2,2.3,4.2s5-2.9,7-4c2.9-1.7,4-4.5,4-4.5s-3.1-5.6-5.9-9.3c-2.9-3.9-7-8.3-8.9-10.3
+		c-3.4-3.3-9.3-7.9-12.3-9.9c-2.7-1.8-7.5-4.4-7.5-4.4s1.1-3.1,1.8-7.7c0.4-2.8,0.6-7.5,0.6-7.5L-105.6,473.2z"/>
+</g>
+</svg>

```

Comment:
This can be losslessly optimized to:

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="-149 451 100 100"><path fill="#dbc01d" d="M-63.7 527l-30.5 17.6a9.4 9.4 0 0 1-9.7 0l-30.5-17.6c-3-1.8-4.8-5-4.8-8.4v-35.2c0-3.5 1.8-6.6 4.8-8.4l30.5-17.6c3-1.8 6.6-1.8 9.7 0l30.5 17.6c3 1.8 4.8 5 4.8 8.4v35.2a9.3 9.3 0 0 1-4.8 8.4z"/><path fill="#383307" d="M-105.6 473.2s.1 5.7-2.2 12.3c-1.6 4.7-3.8 7.1-9 12.5-2.5 2.5-9.5 8.7-14.8 15.4a57.4 57.4 0 0 0-6.4 9.7s.9 2.3 3.7 4l7.4 4.3 3.4-5.6c1.5-2.3 4-5.4 4-5.4l8 12.2 11.5-7.7-9.7-14.5s4.6-4.4 5.8-5.8l3.2-3.7s4.9 2.9 6.8 4.2c2.8 1.8 6.4 4.7 6.4 4.7l-10.1 15.1 11.5 7.7 8.4-12.5 4.7 6.8c1.1 1.7 2.3 4.2 2.3 4.2l7-4c2.9-1.7 4-4.5 4-4.5s-3.1-5.6-5.9-9.3c-2.9-3.9-7-8.3-8.9-10.3-3.4-3.3-9.3-7.9-12.3-9.9-2.7-1.8-7.5-4.4-7.5-4.4s1.1-3.1 1.8-7.7c.4-2.8.6-7.5.6-7.5l-13.7-.3z"/></svg>
```